UserRepository/Tests/UserRepositoryTests.swift

```diff
@@ -0,0 +1,188 @@
+                XCTFail("Unexpected error")
+            }
+        }
+        wait(for: [expectation], timeout: 1)
+        // Then
+        verifyBusinessResponse(user)
+    }
+
+    func testGetUser_failure() {
+        // Given
+        let expectation = self.expectation(description: "Get HyperwalletUser completed")
+        let error = NSError(domain: NSURLErrorDomain, code: 501, userInfo: nil)
+        HyperwalletTestHelper.setUpMockServer(request: UserRequestHelper.setUpRequest(individualUserResponse,
+                                                                                      error))
+        // When
+        userRepository.getUser { result in
+            switch result {
+            case .success:
+                XCTFail("The request should return error!")
+
+            case .failure:
+                expectation.fulfill()
+            }
+        }
+        wait(for: [expectation], timeout: 1)
+    }
+}
+
+private extension UserRepositoryTests {
+    func verifyIndividualResponse(_ user: HyperwalletUser?) {
+        XCTAssertNotNil(user)
+
+        XCTAssertEqual(user!.clientUserId, "myAppUserId01")
+        XCTAssertEqual(user!.token, "YourUserToken")
+        XCTAssertEqual(user!.status, HyperwalletUser.Status.activated)
+        XCTAssertEqual(user!.verificationStatus, HyperwalletUser.VerificationStatus.notRequired)
+        XCTAssertEqual(user!.profileType, HyperwalletUser.ProfileType.individual)
+        XCTAssertEqual(user!.gender, HyperwalletUser.Gender.male)
+        XCTAssertEqual(user!.employerId, "001")
+        XCTAssertNil(user!.countryOfNationality)
+
+        XCTAssertEqual(user!.firstName, "Stan")
+        XCTAssertEqual(user!.middleName, "Albert")
+        XCTAssertEqual(user!.lastName, "Fung")
+        XCTAssertEqual(user!.dateOfBirth, "1980-01-01")
+        XCTAssertEqual(user!.countryOfBirth, "US")
+        XCTAssertEqual(user!.driversLicenseId, "000123")
+        XCTAssertEqual(user!.governmentIdType, "PASSPORT")
+        XCTAssertEqual(user!.passportId, "00000")
+
+        XCTAssertEqual(user!.createdOn, "2019-04-30T00:01:53")
+        XCTAssertEqual(user!.phoneNumber, "000-000000")
+        XCTAssertEqual(user!.mobileNumber, "000-000-0000")
+        XCTAssertEqual(user!.email, "user01@myApp.com")
+        XCTAssertEqual(user!.governmentId, "0000000000")
+
+        XCTAssertEqual(user!.addressLine1, "abc")
+        XCTAssertEqual(user!.addressLine2, "def")
+        XCTAssertEqual(user!.city, "Phoenix")
+        XCTAssertEqual(user!.stateProvince, "AZ")
+
+        XCTAssertEqual(user!.country, "US")
+        XCTAssertEqual(user!.postalCode, "12345")
+        XCTAssertEqual(user!.language, "en")
+        XCTAssertEqual(user!.timeZone, "PST")
+        XCTAssertEqual(user!.programToken, "prg-00000000-0000-0000-0000-000000000000")
+    }
+
+    func verifyBusinessResponse(_ user: HyperwalletUser?) {
+        XCTAssertNotNil(user)
+
+        XCTAssertEqual(user!.clientUserId, "myBusinessIdd01")
```

Comment:
add generic text message to all the asserts

Reply:
like "User token is not as expected"

Reply:
Added generic text messages to all the asserts